File: CS/SpreadWinDemoCS/importexport/opentextfile.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace SpreadWinDemo.importexport
{
    public partial class opentextfile : SpreadWinDemo.DemoBase
    {
        public opentextfile()
        {
            InitializeComponent();

            button1.Click += new EventHandler(button1_Click);
        }

        void button1_Click(object sender, EventArgs e)
        {
            string fn = "";
            // ファイル選択ダイアログ起動
            using (OpenFileDialog ofd = new OpenFileDialog())
            {
                ofd.Filter = "csvファイル(*.csv)|*.csv"; 
                if (ofd.ShowDialog() == DialogResult.OK)
                {
                    fn = ofd.FileName;
                }
                else
                {
                    return;
                }
            }

            // csv読込
            fpSpread1.ActiveSheet.LoadTextFile(fn, FarPoint.Win.Spread.TextFileFlags.ForceCellDelimiter, FarPoint.Win.Spread.Model.IncludeHeaders.ColumnHeadersCustomOnly, System.Environment.NewLine, ",", "\"");
        }
    }
}

File: VB/SpreadWinDemo/importexport/opentextfile.vb
Public Class opentextfile

    Private Sub Button1_Click(sender As Object, e As System.EventArgs) Handles Button1.Click
        Dim fn As String = ""
        ' ファイル選択ダイアログ起動
        Using ofd As New OpenFileDialog()
            ofd.Filter = "csvファイル(*.csv)|*.csv"
            If ofd.ShowDialog() = DialogResult.OK Then
                fn = ofd.FileName
            Else
                Return
            End If
        End Using

        ' csv読込
        FpSpread1.ActiveSheet.LoadTextFile(fn, FarPoint.Win.Spread.TextFileFlags.ForceCellDelimiter, FarPoint.Win.Spread.Model.IncludeHeaders.ColumnHeadersCustomOnly, System.Environment.NewLine, ",", """")
    End Sub
End Class
